src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -318,14 +324,14 @@ public int RenameTrackingPreview
 
         public int ShowKeywords
         {
-            get { return GetBooleanOption(CompletionOptions.IncludeKeywords); }
-            set { SetBooleanOption(CompletionOptions.IncludeKeywords, value); }
+            get { return 0; }
```

Comment:
Agreed that it's not useful if things are coming up on identifiers.  That's why the option was disabled if "show on typed chars" was checked.

This, together with the snippets and show on typed char options were added to allow people used to VS2003 to retain it's completion behavior.  It may be true that there aren't enough people who care anymore to keep the option, but we should take a look at telemetry for VS2013 to verify that before just changing.